src/models/Key.ts

```diff
@@ -0,0 +1,8 @@
+export interface IKey {
```

Comment:
We need more descriptive names... key is sting in our key=>value convention, so lets came up some name that let us know what this interface is for